rasa/core/nlg/generator.py

```diff
@@ -22,7 +22,7 @@ async def generate(
         """Generate a response for the requested template.
 
         There are a lot of different methods to implement this, e.g. the
-        generation can be based on templates or be fully ML based by feeding
+        generation can be based on responses or be fully ML based by feeding
```

Comment:
I think the `template_name` params should be renamed to ``response_name`

Reply:
We talked about this in "person" and we decided on `action_name` since it's already an existing terminology.